app/helpers/content_block_helper.rb

```diff
@@ -0,0 +1,20 @@
+module ContentBlockHelper
+  include Sufia::ContentBlockHelperBehavior
+
+  def displayable_content_block(content_block, **options)
+    return if content_block.value.blank?
+    content_tag :div, content_block.value, options
+  end
+
+  def editable_content_block(content_block)
+    capture do
+      concat form_for([sufia, content_block]) { |f|
```

Comment:
Maybe we should extract the `form_for` into its own helper? That'd also help deal with the block precedence confusion.